vendor/github.com/openshift/library-go/pkg/operator/staticpod/controller/revision/deployment_controller.go

```diff
@@ -106,6 +115,22 @@ func (c RevisionController) createRevisionIfNeeded(operatorSpec *operatorv1.Oper
 		return true, nil
 	}
 
+	if c.validate != nil {
+		if err := c.validate(configs, secrets); err != nil {
+			cond := operatorv1.OperatorCondition{
+				Type:    "RevisionControllerFailing",
+				Status:  operatorv1.ConditionTrue,
+				Reason:  "ContentInvalid",
+				Message: err.Error(),
```

Comment:
that helper function I described to @mfojtik here seems to apply https://github.com/openshift/library-go/pull/150#discussion_r244737541

Reply:
fixed to use `NewMultiLineAggregate`